### PR TITLE
Added check-pr profile to pom.xml for JST

### PR DESCRIPTION
**Pull request by psrna**

### CODE REVIEW

**Comment by psrna** [comment]
@rawagner, please review. Thanks!

File: tests/org.jboss.tools.jst.ui.bot.test/pom.xml [hunk 1]
```diff
@@ -38,4 +40,42 @@
...
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>target-platform-configuration</artifactId>
+						<configuration>
+							<systemProperties>
+								<org.osgi.framework.bundle.parent>ext</org.osgi.framework.bundle.parent>
+							</systemProperties>
+							<dependency-resolution>
+								<extraRequirements>
+									<requirement>
+										<type>p2-installable-unit</type>
```
Comment (rawagner) [inline tests/org.jboss.tools.jst.ui.bot.test/pom.xml]: Are you going to build this feature locally ? If not, than there is no point specifying it as installable unit

File: tests/org.jboss.tools.jst.ui.bot.test/pom.xml [hunk 2]
```diff
@@ -19,6 +18,9 @@
 				<configuration>
 					<testSuite>org.jboss.tools.jst.ui.bot.test</testSuite>
 					<testClass>org.jboss.tools.jst.ui.bot.test.AllJSTTests</testClass>
+					<systemProperties>
+						<org.osgi.framework.bundle.parent>ext</org.osgi.framework.bundle.parent>
+					</systemProperties>
 					<dependencies combine.children="append">
```
Comment (rawagner) [inline tests/org.jboss.tools.jst.ui.bot.test/pom.xml]: you can remove all depedencies here. Doesnt matter if you do mvn verify or specify installBase folder, you will always get full Devstudio

**Comment by rawagner** [comment]
LGTM

File: tests/org.jboss.tools.jst.ui.bot.test/pom.xml [hunk 3]
```diff
@@ -19,23 +18,45 @@
...
+		<profile>
+			<id>check-pr</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>target-platform-configuration</artifactId>
+						<configuration>
+							<systemProperties>
+								<org.osgi.framework.bundle.parent>ext</org.osgi.framework.bundle.parent>
```
Comment (rawagner) [inline tests/org.jboss.tools.jst.ui.bot.test/pom.xml]: you dont need to specify again, its already specified in tycho-surefire-plugin

**Pull request merged**